src/review/routes/dashboard/Dashboard.js

```diff
@@ -49,14 +49,21 @@ function Dashboard(props) {
 
                 return (
                   <List>
-                    {organizations.map(organization => (
+                    {owners.map(({ name }) => (
                       <ListItem
-                        key={organization.id}
+                        key={name}
                         button
                         component={LinkRouter}
-                        to={`/${organization.name}`}
+                        to={`/${name}`}
                       >
-                        <ListItemText primary={organization.name} />
+                        <Layout container align="center">
```

Comment:
You can use an `Avatar` directly https://material-ui-1dab0.firebaseapp.com/#/component-demos/lists
No need for the `Layout`.